src/generated/cli/configv1/datasets.gen.go

```diff
@@ -576,7 +582,7 @@ spec:
                     error:
                         # The value the filter compares to the target trace or span field.
                         value: <true|false>
-        type: <TRACES>
+        type: <TRACES|LOGS>
```

Comment:
In the changelog, can you add a line for 

"Adds support for logs in `v1/config/Dataset`